README.md

```diff
@@ -19,35 +19,28 @@
 
 {% vimeo %}236358264{% endvimeo %}
 
-## How To Get Started
-
-![screenshot of Patchwork](./assets/patchwork.jpg)
-
-The easiest way to get started is using **Patchwork**, an application for navigating and talking on scuttlebutt.
-
-1. [Download Patchwork](http://dinosaur.is/patchwork-downloader/) for Windows, macOS and Linux
-2. Get a [pub invite code](https://github.com/ssbc/scuttlebot/wiki/Pub-Servers)
-3. Use the pub invite: in Patchwork click `+ Join Pub` and paste the invite code
-
-Step 2 is important if you want to be able to find people further than your local network - it sets you up with a robot friend on the internet which helps you find and communicate with people.
-Read more about Pubs and their role [here](./concepts/pub.md).
-
-## Join some communities
-
-Once onboard patchwork, check out the following channels by typing their name (with #) in the search bar:
-- **#new-people**: introduce yourself
-- **#faq**: first impressions, what is confusing as a new user?
-- **#patchwork**: report bugs, suggestions, etc
-- **#scuttlebutt**: protocol related discussion
-
-Here are some other favourites:
-- **#boats**: people living on, hacking, and building boats
-- **#cooking-channel**: nerds sharing recipes
-- **#solarpunk**: step into the future
-- **#gardening**: get some gardening on with your off-grid
-- **#anarchitecture**: decentralise all the things
-
-## Code of conduct
+## Ready to get started?
+**We have a detailed [Getting Started Guide](getting_started.md) which is
+recommended if you've got a few minutes.**
+
+If you just need to get up and running ASAP here's what you need to know:
+
+### Quick Start
+
+1. [Download Patchwork](http://dinosaur.is/patchwork-downloader/), and install
+   it like any other app. 
+2. Boot it up and set up your profile. 
+3. Go to [the list of pub
+   servers](https://github.com/ssbc/scuttlebot/wiki/Pub-Servers) and get an
+   invite code from one.
+4. Click "+ Join Pub" in the top left corner of Patchwork and paste in the
+   invite code.  It's best to have multiple pub servers to extend your
+   visibility into the network. 
+5. Start browsing public posts and channels. Note that there's a "More Channels"
```

Comment:
do we want to say anything about following new people? i noticed that some newbies have been subscribing to many channels, then being confused why they are mostly empty.

i guess i mean to say: i'd personally rather we talk about following people than channels.

Reply:
adding a line between 5 and 6 about following people